src/main/java/com/marek/onlinebookstore/model/OrderItem.java

```diff
@@ -0,0 +1,42 @@
+package com.marek.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@SQLDelete(sql = "UPDATE order_items SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted = false")
+@Table(name = "order_items")
+public class OrderItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(nullable = false, name = "order_id")
+    private Order order;
+    @OneToOne
+    @JoinColumn(nullable = false, name = "book_id")
+    private Book book;
+    @Column(nullable = false)
+    private int quantity;
+    @Column(nullable = false)
+    private BigDecimal price;
+    @Column(name = "is_deleted",nullable = false)
```

Comment:
The 'isDeleted' field should not be a part of the business model, but rather be handled by JPA auditing fields.